GVFS/GVFS.Platform.Linux/GVFS.Platform.Linux.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\GVFS.Build\GVFS.cs.props" />
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1;netstandard2.0</TargetFrameworks>
+    <Platforms>x64</Platforms>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
I didn't see any code marked as `unsafe` in `GVFS.Platform.Linux`, can `<AllowUnsafeBlocks>true</AllowUnsafeBlocks>` be removed?

Reply:
Yes, removed!